### ISSUE TITLE
Issue with app bundle

### ISSUE THREAD

**Issue by vishalknishad** [dreamsoftin/facebook_audience_network#109]
I created an app bundle and published to google play, the ads are not showing up.
However when I create a release apk and send to my device directly, ads are showing fine.

**Comment by SachinGanesh** [comment]
In release builds, it may take up to 24 hours to show ads

**Comment by vishalknishad** [comment]
It's been more than 24 hours since update.

**Comment by vishalknishad** [comment]
It has been 3 days, still ads not showing. Some help will be appreciated.

**Comment by vishalknishad** [comment]
Hey, can someone help please. Ad requests are coming but ads are not showing at all.
![FAN](https://user-images.githubusercontent.com/34787387/123203348-d617bd00-d4d3-11eb-8e51-51a83b0cceac.png)

**Comment by rojapeiris** [comment]
I had faced the same issue.  I noticed that SDK integration is not green colored.
![Capture](https://user-images.githubusercontent.com/27843813/123210759-19782880-d4e0-11eb-9268-cecdcccdad74.PNG)

**Comment by vishalknishad** [comment]
@rojapeiris for me, all are green

**Comment by rojapeiris** [comment]
hi dear,
In my case facebook ada are showing properly but no any impressions. But in
the property section sdk integration dosnt appear green colored.

**Comment by adarshhub** [comment]
Same problem here. 
In my case, the app is approved and the ads are showing on the test device. 
But in FAN it has been showing 'waiting for ad request' for more than a week.

**Comment by aliakkawi** [comment]
> Same problem here.
> In my case, the app is approved and the ads are showing on the test device.
> But in FAN it has been showing 'waiting for ad request' for more than a week.

same hear, i am unable to state the issue, is it something related to the code implementation  or Placements creation

**Comment by aliakkawi** [comment]
Same here, i am unable to state the issue, is it something related to the code implementation or Placements creation, any solution?

**Comment by adarshhub** [comment]
No, it is not related to the integration of the ads because the test ads are coming on the app downloaded from the play store.

I think this is related to the review of the ads. 

The Facebook team is not reviewing the app. Maybe, because the app is not requesting ads through bidding.

If that is the case, then this plugin is redundant for the new apps.

**Comment by aliakkawi** [comment]
> No, it is not related to the integration of the ads because the test ads are coming on the app downloaded from the play store.
> 
> I think this is related to the review of the ads.
> 
> The Facebook team is not reviewing the app. Maybe, because the app is not requesting ads through bidding.
> 
> If that is the case, then this plugin is redundant for the new apps.

I've contacted Facebook team and they told me that Bidding is not a must for now at least for Android.

**Comment by aliakkawi** [comment]
@SachinGanesh Can you please advice us what to do, i've set my device as a Test device, and i can see real ads even when i download my Android app from Google play store, but no ads are getting displayed for other users who download app from the store, it 've released my app to the store more than one week ago.
Thanks

**Comment by vishalknishad** [comment]
Hey guys, I just realised that FAN is switched to bidding only. It means, you have to use mediation platforms to show ads now.

I did mediation with Admob and ads are showing up now.

I used following package https://pub.dev/packages/native_admob_flutter

Mediation guide https://developers.google.com/admob/android/mediation/facebook

**Comment by aliakkawi** [comment]
> Hey guys, I just realised that FAN is switched to bidding only. It means, you have to use mediation platforms to show ads now.
> 
> I did mediation with Admob and ads are showing up now.
> 
> I used following package https://pub.dev/packages/native_admob_flutter
> 
> Mediation guide https://developers.google.com/admob/android/mediation/facebook

@vishalknishad Thanks for the update, you are using native_admob_flutter instead of facebook_audience_network or you are using both?

**Comment by vishalknishad** [comment]
@aliakkawi native_admob_flutter only

**Comment by aliakkawi** [comment]
> @aliakkawi native_admob_flutter only

Hello @vishalknishad After using "native_admob_flutter" i can finally see both AdMob and Facebook ads in my app, 
but the issue is that match rate for Facebook is very low in comparison to Admob, also the status for Facebook placements are still "Requesting ads", [
![WhatsApp Image 2021-07-07 at 16 38 11](https://user-images.githubusercontent.com/14969941/124769099-e0a96c00-df41-11eb-8e67-911a8f0ccfe7.jpeg)
![WhatsApp Image 2021-07-07 at 16 36 45](https://user-images.githubusercontent.com/14969941/124769105-e2732f80-df41-11eb-93a6-cfbb023ee45d.jpeg)
](url) are you facing the same issue? Please check the attached screen shots

**Comment by lostidol355** [comment]
> Hey guys, I just realised that FAN is switched to bidding only. It means, you have to use mediation platforms to show ads now.
> 
> I did mediation with Admob and ads are showing up now.
> 
> I used following package https://pub.dev/packages/native_admob_flutter
> 
> Mediation guide https://developers.google.com/admob/android/mediation/facebook

Hi I am using google mobile ads plugin but ineterstial ads is not showing through bidding , can you guide me why?

**Comment by vishalknishad** [comment]
@lostidol355 did you followed the mediation guide?
Also, check the error response you are getting while loading the ads.

**Comment by lostidol355** [comment]
> @lostidol355 did you followed the mediation guide?
> Also, check the error response you are getting while loading the ads.

Yes sir I have followed mediation guide correctly. Bidding is successfully happening for banner ads. But I do not get any bid request for interstial ads. I am able to show facebook interstial ads through waterfall (  without using this plugin code,)  but I am not getting any bidrequest for inetrstial ads.  Please help me sir I am beginer too
